### ISSUE TITLE
Incorrect indentation inside directives

### ISSUE THREAD

**Issue by joaopms** [onecentlin/laravel-blade-snippets-vscode#92]
Formatting a Blade document is incorrectly indenting code inside directives.

**Expected output:**
```blade
@section('content')
    <p>Hello world</p>
@endsection
```

**Actual output:**
```blade
@section('content')
<p>Hello world</p>
@endsection
```

**Comment by Stevemaster92** [comment]
Similar with `@for` directives.

OS: Windows 10, Linux Fedora 29
VS Code: 1.33.1
Laravel Blade Snippets: 1.20.0

**Expected output:**
```blade
<select class="form-control custom-select" name="week">
    @for ($i = 0; $i < 52; $i++)
        <option value="{{ $i }}">Week {{ $i+1 }}</option>
    @endfor
</select>
```

**Actual output:**
```blade
<select class="form-control custom-select" name="week">
    @for ($i = 0; $i < 52; $i++) <option value="{{ $i }}">Week {{ $i+1 }}</option>
        @endfor
</select>
```

**Comment by IsidroMar95** [comment]
does anyone solved it? i have the same issue too and i'm using phpStorm meanwhile :/

**Comment by andrewmclagan** [comment]
Yeah its a hard issue to deal with, anyone have any other blade formatting plugins?

**Comment by MartinP7r** [comment]
I'd love for this to get implemented. Is anybody still maintaining this project?

**Comment by Slivicon** [comment]
also discussed in #61

**Comment by elliottregan** [comment]
🦗 🦗 🦗

**Comment by alexkerber** [comment]
:/ any other solutions than this?

**Comment by tsekka** [comment]
> :/ any other solutions than this?

You can use beautifier. This gist has some bugs but with some effort you can make it work. Look at the comments below the gist.


https://gist.github.com/fzldn/a27973ff7e4c8e3738b0e06e525f7403

**Comment by onecentlin** [comment]
> You can use beautifier. This gist has some bugs but with some effort you can make it work. Look at the comments below the gist.
> 
> https://gist.github.com/fzldn/a27973ff7e4c8e3738b0e06e525f7403

@tsekka Thank you for sharing. For now, it seems a better solution for indentation formatting.
Beautifier converts all blade directives to html tag format in order to process document like html.

**Comment by alexkerber** [comment]
> > :/ any other solutions than this?
> 
> You can use beautifier. This gist has some bugs but with some effort you can make it work. Look at the comments below the gist.
> 
> https://gist.github.com/fzldn/a27973ff7e4c8e3738b0e06e525f7403

Still not a solution in my eyes that you have to change this every time they update the plugin. :/

**Comment by rsdsoft** [comment]
Broken for years lol

**Comment by kylemilloy** [comment]
https://github.com/shufo/vscode-blade-formatter

**Comment by jerothor** [comment]
Since this is still broken, you can use the [Laravel Blade Formatter](https://marketplace.visualstudio.com/items?itemName=shufo.vscode-blade-formatter) package instead of this for formatting.

And be sure to disable the formatting from this package in your settings:

```json
"blade.format.enable": true,
```
